Demo/src/main/java/com/braintreepayments/demo/PayPalFragment.java

```diff
@@ -46,8 +46,8 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
         braintreeClient = getBraintreeClient();
         payPalClient = new PayPalClient(braintreeClient);
         payPalLauncher = new PayPalLauncher(
-                payPalBrowserSwitchResult -> payPalClient.onBrowserSwitchResult(
-                        payPalBrowserSwitchResult, (payPalAccountNonce, error) -> {
+                payPalPaymentAuthResult -> payPalClient.tokenize(
```

Comment:
```suggestion
                payPalAuthResult -> payPalClient.tokenize(
```
nit: Maybe we can make it more concise by removing the word `payment`?

Reply:
Do you mean rename the entire class or just the variables in these callbacks? I renamed the variables to `paymentAuthResult` to make it more concise. I think `AuthResult` might be a little too generic for the types but open to opinions on that.

Reply:
Yeah I guess the overall naming convention in general to be more specific. I get that adding the word payment may make it more descriptive though. Total take or leave.

I've got the [Swift naming guidelines](https://www.swift.org/documentation/api-design-guidelines/#promote-clear-usage) in mind. I find it to be a useful grammatical guide to naming in any language even though it's geared towards Swift.